src/Number/index.ts

```diff
@@ -46,3 +46,33 @@ export declare function divideWithModulo(
   a: number,
   b: number,
 ): readonly [number, number]
+
+/** Returns `true` if the first argument is greater than second argument. Otherwise it returns `false` */
+
+export declare function gt<A>(b: A): (a: A) => boolean
+
+export declare function gt<A>(a: A, b: A): boolean
+
+/** Returns `true` if the first argument is greater or equal than second argument. Otherwise it returns `false` */
+
+export declare function gte<A>(b: A): (a: A) => boolean
+
+export declare function gte<A>(a: A, b: A): boolean
+
+/** Returns `true` if the first argument is lower than second argument. Otherwise it returns `false` */
+
+export declare function lt<A>(b: A): (a: A) => boolean
+
+export declare function lt<A>(a: A, b: A): boolean
+
+/** Returns `true` if the first argument is lower or equal than second argument. Otherwise it returns `false` */
+
+export declare function lte<A>(b: A): (a: A) => boolean
+
+export declare function lte<A>(a: A, b: A): boolean
```

Comment:
@domeknn I posted a few code suggestions above, I think `a` and `b` shouldn't be a generic type, and instead, we should define argument types explicitly